sdk/swagger.json

```diff
@@ -1065,6 +1108,17 @@
                     "items": {
                         "$ref": "#/definitions/Datasource"
                     }
+                },
+                "defaultMode": {
```

Comment:
I want.
Create dataset is there long time ago.
I am just adding a defaultMode. which is also exists a long time ago